Lib/sysconfig.py

```diff
@@ -635,8 +640,9 @@ def get_config_vars(*args):
     With arguments, return a list of values that result from looking up
     each argument in the configuration variable dictionary.
     """
-    global _CONFIG_VARS
-    if _CONFIG_VARS is None:
+
+    def init_config_vars():
```

Comment:
I think we can move this function outside, to decrease the complexity of this function.